### PR TITLE
fix: resnet_bpda baseline not set to eval mode

### PR DESCRIPTION
**Pull request by andrewwhitecdw**
### Problem
fix: resnet_bpda baseline not set to eval mode

### Fix
Replace:
```
    resnet_bpda = ResNet_Adv_Model(args, config)
    if ngpus > 1:
        resnet_bpda = torch.nn.DataParallel(resnet_bpda)

    start_time = time.time()
    init_acc = get_accuracy(resnet_bpda, x_val, y_val, bs=adv_batch_size)
```

with:
```
    resnet_bpda = ResNet_Adv_Model(args, config).eval()
    if ngpus > 1:
        resnet_bpda = torch.nn.DataParallel(resnet_bpda)

    start_time = time.time()
    init_acc = get_accuracy(resnet_bpda, x_val, y_val, bs=adv_batch_size)
```

### Files changed
- `eval_sde_adv_bpda.py`

### CODE REVIEW

**Comment by andrewwhitecdw** [comment]
Closing this sweep-generated PR: sole commit is missing a valid Signed-off-by trailer. It does not meet the sweep requirements (single signed-off commit).
